samples/cpp/visual_language_chat/visual_language_chat.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "load_image.hpp"
+#include <openvino/genai/vlm_pipeline.hpp>
+#include <openvino/runtime/intel_gpu/properties.hpp>
+
+bool print_subword(std::string&& subword) {
+    return !(std::cout << subword << std::flush);
+}
+
+int main(int argc, char* argv[]) try {
+    if (3 != argc) {
+        throw std::runtime_error(std::string{"Usage "} + argv[0] + " <MODEL_DIR> <IMAGE_FILE>");
```

Comment:
```suggestion
        throw std::runtime_error(std::string{"Usage "} + argv[0] + " <MODEL_DIR> <PROMPT> <IMAGE_FILE>");
```

Reply:
Current version follows `chat_sample` approach. Prompts are going to be read anyway by `cin`.

Reply:
agree, missed this part